docs/content/concepts/partitions-schedules-sensors/sensors.mdx

```diff
@@ -334,6 +394,34 @@ def test_my_directory_sensor_cursor():
         assert validate_run_config(log_file_job, run_request.run_config)
 ```
 
+#### Testing sensors with resources
+
+For sensors which utilize [resources](/concepts/resources), you can provide the necessary resources when invoking the sensor function.
+
+Below is a test for the `process_new_users_sensor` that we defined [above](#using-resources-in-sensors) which uses the `users_api` resource.
```

Comment:
I think building on previous examples is great, but I don't like that I have to click the link to figure out what example you're referring to. Perhaps something like:

> The following is a test for the `process_new_users_sensor` defined in the [Using resources in sensors](#using-resources-in-sensors) section, which uses the `users_api` resource: